### PR TITLE
Final Rubocop clean up after bye bye spree

### PR DESCRIPTION
**Pull request by luisramos0**
#### What? Why?

This makes rubocop issue no warnings when executed. The exceptions files are now up to date with all the file moves from bye bye spree.

Relates to #6055 

And in the final commit I add the frozen_string_literal to all specs.

#### What should we test?
Green build.

#### Release notes
Changelog Category: Technical changes
Fixed rubocop exception files to match the current files in the codebase.

### CODE REVIEW

**Comment by Matt-Yorkley** [comment]
I think we should consider outsourcing this job to a bot in the future: https://github.com/marketplace/rubocop-auto-fixer

**Comment by luisramos0** [comment]
I dont mind the 10 minutes of mechanical work every now and then...

**Comment by sauloperez** [comment]
wow! that bot is amazing!

**Comment by Matt-Yorkley** [comment]
Shall we try adding the bot? Now that Luis is leaving nobody will ever bother doing this task... :see_no_evil:

**Comment by sauloperez** [comment]
I'm totally up for it.

File: engines/web/lib/web/version.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Web
-  VERSION = "0.0.1".freeze
+  VERSION = "0.0.1"
```
[the remark this answers is not in the document]
Comment (luisramos0) [reply engines/web/lib/web/version.rb:4]: This bot is still not picking up all our rules. We dont have this rule in OFN.

**Review by mkllnk — APPROVED** [review]
Nice work. I ran rubocop locally and still got 6 offenses but that's heaps better than before. :+1:

**Review by sauloperez — APPROVED** [review]

**Comment by sauloperez** [comment]
Does someone disagree if I merge this right away @Matt-Yorkley @andrewpbrett ?

**Pull request merged**